app/index.html

```diff
@@ -40,7 +40,12 @@
       <div class="meniu"><i class="fa fa-2x fa-bars"></i></div>
     </header>
     <main>
-      <section id="welcome-section"></section>
+      <section id="welcome-section">
+        <h1 class="welcome-section-left">
```

Comment:
in this case left is a modifier, because you use welcome-section and left describes that it is aligned to left. So modifiers should have -- between main element. In your case class name should be welcome-section--left